frontend/views/components/Avatar.vue

```diff
@@ -77,8 +79,8 @@ export default {
   },
   watch: {
     blobURL (newBlobURL, oldBlobURL) {
-      console.debug(`Avatar blobURL watcher: ${oldBlobURL} => ${newBlobURL}`)
-      this.setBlobURL(newBlobURL)
+      console.log(`Avatar blobURL watcher: ${oldBlobURL} => ${newBlobURL}`)
+      this.setBlobURL(newBlobURL, oldBlobURL)
```

Comment:
Yeah, agreed with @pieer, not sure why this code is being modified. The old code worked fine I think (in terms of getting `oldBlobURL`). Also, why change `.debug` to `.log`? Unless there is a reason for it, please keep it as `.debug` since it is just debug output and nothing important.